certified-connectors/RescoReports/apiDefinition.swagger.json

```diff
@@ -0,0 +1,400 @@
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ReportId.Optional"
+          },
+          {
+            "$ref": "#/parameters/List.Format"
+          }
+        ],
+        "responses": {
+          "200": {
+            "$ref": "#/responses/Report.File"
+          },
+          "500": {
+            "$ref": "#/responses/error"
+          }
+        }
+      }
+    },
+    "/gen-multi-report": {
+      "get": {
+        "operationId": "GenMultiReport",
+        "summary": "Generate a report on multiple rows",
+        "description": "Generates a report for provided list of table rows.",
+        "produces": [
+          "application/pdf",
+          "text/html",
+          "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+          "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+          "text/plain"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ReportId"
+          },
+          {
+            "in": "query",
+            "name": "sources",
+            "description": "Comma-delimited list of record IDs",
+            "x-ms-summary": "Comma-delimited list of record IDs",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/List.Format"
+          }
+        ],
+        "responses": {
+          "200": {
+            "$ref": "#/responses/Report.File"
+          },
+          "500": {
+            "$ref": "#/responses/error"
+          }
+        }
+      }
+    },
+    "/gen-fetch-report": {
+      "get": {
+        "operationId": "GenFetchReport",
+        "summary": "Generate a report on multiple rows (fetch XML)",
+        "description": "Generates a report for a set of table rows defined by fetch XML query.",
+        "produces": [
+          "application/pdf",
+          "text/html",
+          "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+          "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+          "text/plain"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ReportId"
+          },
+          {
+            "in": "query",
+            "name": "sourcefetch",
+            "description": "Fetch XML request defining source entities",
+            "x-ms-summary": "Fetch XML request defining source entities",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/List.Format"
+          }
+        ],
+        "responses": {
+          "200": {
+            "$ref": "#/responses/Report.File"
+          },
+          "500": {
+            "$ref": "#/responses/error"
+          }
+        }
+      }
+    },
+    "/resco_mobilereports": {
+      "get": {
+        "summary": "List reports",
+        "description": "Get list of records",
+        "operationId": "Report-List",
```

Comment:
@JuroJ all operationIds should be in Camel Case. Please chagne to "ReportList".
Please fix all operationIds in the swagger file.

Reply:
Done